pages/auditlog/setup.rst

```diff
@@ -0,0 +1,248 @@
+*****
+Setup
+*****
+
+The Audit Log plugin is a commercial Graylog feature that can be installed in
+addition to the Graylog open source server.
+
+Installation
+============
+
+Please see the :doc:`Graylog Enterprise setup page </pages/enterprise/setup>` for details on how to install
+the Audit Log plugin.
+
+.. note:: Make sure the Audit Log plugin is installed on every node in your Graylog cluster.
+
+Configuration
+=============
+
+The audit log plugin provides two ways of writing audit log entries:
+
+1. Database
+2. Log file via `log4j2 <https://logging.apache.org/log4j/2.x/>`_ appender
+
+Logging to the database is always enabled and cannot be disabled.
+
+All configuration needs to be done in the Graylog server configuration file
+and in the logging configuration. (only if the log4j2 appender is enabled)
+
```

Comment:
this should be a *hint* cause it will not be noticed and a link to the default file locations: http://docs.graylog.org/en/2.1/pages/configuration/file_location.html